Modules/Filtering/ImageGrid/test/itkResampleImageTest7.cxx

```diff
@@ -0,0 +1,170 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include <iostream>
+
+#include "itkAffineTransform.h"
+#include "itkResampleImageFilter.h"
+#include "itkPipelineMonitorImageFilter.h"
+#include "itkStreamingImageFilter.h"
+#include "itkTestingMacros.h"
+#include "itkMath.h"
+
+/* itkResampleImageFilter output compared to streamed output
+ */
+
+int itkResampleImageTest7( int argc, char * argv [] )
+{
+
+  if (argc > 1)
+    {
+    std::cout << "Usage: " << argv[0]
+              << " noParams" << std::endl;
+    return EXIT_FAILURE;
+    }
```

Comment:
May be add a message saying that arguments are missing, as recommended in `C.23.3 Arguments in Tests` of the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf), and as done in the other test contributed.

Reply:
removed in https://github.com/romangrothausmann/ITK/commit/265f7bb50221495fc6e1fe7f021dbd5ba2700516#diff-dfa5983cf67253ce0e03aca56a207db2L47